src/game/Credits.cc

```diff
@@ -440,6 +444,12 @@ static UINT32 GetNumber(const wchar_t* const string)
 	return v;
 }
 
+static UINT32 GetColor(const wchar_t* const string)
+{
+	const unsigned int v = GetNumber(string);
+	return v < 256 ? gColorFromPalette[v] : RGB(255, 0, 0);
```

Comment:
Please add a logging call here: we don't want anyone to just rely on getting red this way. Or if something becomes erroneously red, it will be faster to find the culprit.
I got bit by a similar bug in gemrb, where palette sizes can be different and we were just silently padding with black.

Reply:
Will add Assert()... that code is called only from credits screen and don't think anyone will use it. Integer v is colormap index so it's never going to be >255. Will add assert in case someone decides to call it with bigger value.